cherrypy/wsgiserver/__init__.py

```diff
@@ -2052,11 +2052,13 @@ def bind(self, family, type, proto=0):
         if self.ssl_adapter is not None:
             self.socket = self.ssl_adapter.bind(self.socket)
 
+        host, port = self.bind_addr[:2]
```

Comment:
`host = self.bind_addr[0]` would be better, since port is not used in this method